.ci/deps.sh

```diff
@@ -67,3 +69,13 @@ julia -e "Pkg.add(\"Lint\")"
 
 # Lua commands
 sudo luarocks install luacheck --deps-mode=none
+
+# Infer commands
+wget -nc -O ~/infer.tar.xz https://github.com/facebook/infer/releases/download/v0.7.0/infer-linux64-v0.7.0.tar.xz
+tar xf ~/infer.tar.xz -C ~/
+cd ~/infer-linux64-v0.7.0
+opam init --y
+opam update
+opam pin add --yes --no-action infer .
+opam install --deps-only --yes infer
+./build-infer.sh java
```

Comment:
Just a reminder about that script - I think it'd make things easier ? 
If you think this is fine - let me know